### ISSUE TITLE
XML requests

### ISSUE THREAD

**Issue by nnevill** [giatro/xml-rpc-js-client#1]
Is it possible to send request in XML format but not in json?

**Comment by giatro** [comment]
No, json request its not supported; I can add support for json requests on a future release.

**Comment by nnevill** [comment]
But I don't need json I need XML... What about XML requests?

**Comment by giatro** [comment]
I will add XML format for params in next release.

**Comment by sheeldev** [comment]
Hello, the extension is very good and easy to use. i just wish i could change the encoding of the response. at the moment it is so-8859-1 and didn't find a way to change it
